src/node/blockstorage.cpp

```diff
@@ -10,6 +10,7 @@
 #include <flatfile.h>
 #include <hash.h>
 #include <kernel/chainparams.h>
+#include <kernel/context.h>
```

Comment:
This must have snuck in during one of the revisions.

Reply:
re: https://github.com/bitcoin/bitcoin/pull/28051#discussion_r1304822985

> This must have snuck in during one of the revisions.

Thanks, reverted